formats/tabular-converter-plugin/src/polus/plugins/formats/tabular_converter/tabular_converter.py

```diff
@@ -0,0 +1,180 @@
+"""Tabular Converter."""
+import enum
+import logging
+import os
+import pathlib
+
+import fcsparser
+import vaex
+
+logger = logging.getLogger(__name__)
+
+POLUS_TAB_EXT = os.environ.get("POLUS_TAB_EXT", ".arrow")
+
+
+class Extension(str, enum.Enum):
+    """Extension types to be converted."""
+
+    FITS = ".fits"
+    FEATHER = ".feather"
+    PARQUET = ".parquet"
+    HDF = ".hdf5"
+    FCS = ".fcs"
+    CSV = ".csv"
+    ARROW = ".arrow"
+    Default = "default"
+
+
+class Convert_tabular:
```

Comment:
Class and type names should not have underscores. Use `ConvertTabular` instead. Look up Python's style guide for class name formatting for details.